src/python/doc/persistence_graphical_tools_user.rst

```diff
@@ -75,9 +91,9 @@ If you want more information on a specific dimension, for instance:
         only_this_dim=1)
     pers_diag = [(1, elt) for elt in birth_death]
     # Use subplots to display diagram and density side by side
-    fig, axes = plot.subplots(nrows=1, ncols=2, figsize=(12, 5))
+    fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(12, 5))
     gudhi.plot_persistence_diagram(persistence=pers_diag,
         axes=axes[0])
     gudhi.plot_persistence_density(persistence=pers_diag,
         dimension=1, legend=True, axes=axes[1])
```

Comment:
Maybe remove the greyblock on density (nicer I found) and it will be a use case of this functionnality
```suggestion
    gudhi.plot_persistence_density(persistence=pers_diag,
        dimension=1, legend=True, greyblock=True, axes=axes[1])
```

Reply:
Do you mean set the Default to `False` ?

Reply:
My point was only for this particular example, but as you want:
* only for this example
* or default greyblock value is False for density
On this particular example, some colors are hidden by the greyblock.

Reply:
Ok I get it, maybe it is better with density to set greyblock to `False` by default, as indeed, a density is not requested to be located above the diagonal so the greyblock is not especially meaningful.

Reply:
I didn't follow or test, but is the grey block drawn above or below the density, if both try to paint the same point? (probably related to the order in which they appear in the function)

Reply:
It is printed above. I think it is just nice to left this option accessible but safer to set it to False by default.

Reply:
Whether it is True of False by default, would printing it below look nicer?

Reply:
I don't think it makes sense this way: when the density is printed, is overlap the whole square (so it would erase the grey block if the latter is printed before). Furthermore, the density is a gaussian KDE, so it never equals zero ; we would have to pick a threshold (or a `alpha` opacity parameter) and I don't think the rendering would be satisfactory in any case.

Also, it is a matter of taste, but I prefer to have the grey block above so that you clearly see the diagonal (I don't see the point of printing a grey block if below, actually).

Reply:
thanks for the explanations.